packages/rum/src/domain/record/serializationUtils.spec.ts

```diff
@@ -95,3 +101,89 @@ describe('getElementInputValue', () => {
     })
   })
 })
+
+describe('Build stylesheet absolute url', () => {
+  const cssHref = 'https://site.web/app-name/static/assets/resource.min.css'
+  const resolvedPath = 'https://site.web/app-name/static/assets/fonts/fontawesome-webfont.eot'
+
+  describe('convert relative path to absolute', () => {
+    it('should replace url when wrapped in single quote', () => {
+      const cssText = "{ font-family: FontAwesome; src: url('./fonts/fontawesome-webfont.eot'); }"
+      expect(switchToAbsoluteUrl(cssText, cssHref)).toEqual(
+        `{ font-family: FontAwesome; src: url('${resolvedPath}'); }`
+      )
+    })
+    it('should replace url when wrapped in double quote', () => {
+      const cssText = '{ font-family: FontAwesome; src: url("./fonts/fontawesome-webfont.eot"); }'
+      expect(switchToAbsoluteUrl(cssText, cssHref)).toEqual(
+        `{ font-family: FontAwesome; src: url("${resolvedPath}"); }`
+      )
+    })
+    it('should replace url when not in any quote', () => {
+      const cssText = '{ font-family: FontAwesome; src: url(./fonts/fontawesome-webfont.eot); }'
+
+      expect(switchToAbsoluteUrl(cssText, cssHref)).toEqual(`{ font-family: FontAwesome; src: url(${resolvedPath}); }`)
+    })
+    it('should replace url when path is relative', () => {
+      const cssText = '{ font-family: FontAwesome; src: url(fonts/fontawesome-webfont.eot); }'
+
+      expect(switchToAbsoluteUrl(cssText, cssHref)).toEqual(`{ font-family: FontAwesome; src: url(${resolvedPath}); }`)
+    })
+    it('should replace url when path is at parent level', () => {
+      const cssText = "{ font-family: FontAwesome; src: url('../fonts/fontawesome-webfont.eot'); }"
+
+      expect(switchToAbsoluteUrl(cssText, cssHref)).toEqual(
+        "{ font-family: FontAwesome; src: url('https://site.web/app-name/static/fonts/fontawesome-webfont.eot'); }"
+      )
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏for consistentcy sake
```suggestion
      expect(switchToAbsoluteUrl(cssText, cssHref)).toEqual(`{ font-family: FontAwesome; src: url(${resolvedPath}); }`)
```

Reply:
For this one, it is not the same value as `resolvedPath`, since the url to convert goes up to the parent (`../`)

Reply:
my bad 🤦‍♂️